### PR TITLE
added how to render a screen 'learn/contributing/plugins'

### PR DESCRIPTION
**Pull request by blikq**
Thought this would be a great addition to the rust bevy book.

### CODE REVIEW

File: content/learn/book/getting-started/plugins/_index.md [hunk 1]
```diff
@@ -35,6 +35,37 @@ You should hopefully notice two things:
...
+The output of everything done to this point was only on console. But ofcourse bevy isn't restricted to that so, let's render a window. Change your importations to this:
+
+```rs
+use bevy::{prelude::*, window::*};
+```
+
+Then edit your `main` function with the following:
+
+```rs
+fn main() {
```
Comment (cart) [inline content/learn/book/getting-started/plugins/_index.md:48]: I think this type of "window configuration" belongs in a dedicated window section. This brings in too many windowing details too early in the learning path. This page should just have the minimal details required to define a plugin.

Comment (cart) [reply content/learn/book/getting-started/plugins/_index.md:48]: Probably best to hold off until we start working on the next book version. @alice-i-cecile any thoughts here?

**Comment by alice-i-cecile** [comment]
Definitely a useful thing to teach about, agreed on "not quite the right learning path". Let's hold off: we can cover this in a section about configuring windows.

**Comment by blikq** [comment]
Ok, noted

**Pull request closed without merging**